website/public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /
```

Comment:
Why do we want to disallow robots from indexing out site? This is the *opposite* of what we want! I want quick-lint-js.com to appear on DuckDuckGo and other search engines.

Or am I misunderstanding what `Disallow: /` means in this context?